tools/scylla-nodetool.cc

```diff
@@ -3435,6 +3534,53 @@ For more information, see: {}
                 clearsnapshot_operation
             }
         },
+        {
+            {
+                "cluster",
+                "Runs operations that affect the entire cluster",
+                "",
+                { },
+                {
+                    typed_option<sstring>("command", "The name of the subcommand", 1),
+                },
+                {
+                    {
+                        "repair",
+                        "Synchronize data between nodes in the background",
+fmt::format(R"(
+When running nodetool cluster repair on a single node, all tablets of
+the specified keyspace(s) are repaired, even if they have no replica on
+this node.
+
+To repair all of the data in the cluster, it is enough to run
+nodetool cluster repair on one node only.
+
+Note that nodetool cluster repair repairs only tablet keyspaces.
+To repair vnode keyspaces use nodetool repair.
+
+For more information, see: {}"
+)", doc_link("operating-scylla/nodetool-commands/cluster/repair.html")),
+                        {
+                            typed_option<std::vector<sstring>>("in-dc", "Constrain repair to specific datacenter(s)"),
+                            typed_option<>("in-local-dc", "Constrain repair to the local datacenter only"),
```

Comment:
We can archive the -in-local-dc option with -in-dc option. The in-local-dc will issue different repairs when the command runs on different node in the cluster, which is bad and error-prone. I would recommend drop in-local-dc option.